CHANGELOG.md

```diff
@@ -87,6 +106,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Other
 
+- Added `examples/daily-multi-translation` to showcase how to send multiple
```

Comment:
Do we have any examples for receiving custom tracks, audio or video ? 
Maybe we can create that in a follow up PR.

Reply:
I have one, yes. I will add it.

Reply:
Added it!